projects/net-snmp/Dockerfile

```diff
@@ -21,5 +21,4 @@ WORKDIR net-snmp
 COPY build.sh $SRC/
 #
 # Until the project moves the fuzzers to the source tree
-COPY snmp_pdu_parse_fuzzer.c $SRC/
-COPY agentx_parse_fuzzer.c $SRC/
+COPY *_fuzzer.c $SRC/
```

Comment:
Where does this change come from? If this is a change introduced by you please add your copyright at the top instead of updating the Google copyright year.

Reply:
This is added because 2 more fuzzers have been added thus it's easier to copy all of them in with one `COPY`. The copyright is needed to satisfy the terms of OSS-Fuzz - also am collaborating with the Google folks in the OSS-Fuzz team and the license is needed as is

Reply:
Removed copyright updates on any of the files that were modified now - only added the copyright headers in the new fuzzers.